packages/headless/src/controllers/commerce/context/cart/headless-cart.ssr.ts

```diff
@@ -1,11 +1,11 @@
-import {SharedControllerDefinitionWithoutProps} from '../../../../app/commerce-ssr-engine/types/common';
+import {SearchAndListingControllerDefinitionWithoutProps} from '../../../../app/commerce-ssr-engine/types/common';
 import {Cart, buildCart, CartProps, CartInitialState} from './headless-cart';
 
 export type {CartState, CartItem, CartProps} from './headless-cart';
 export type {Cart, CartInitialState};
 
 export interface CartDefinition
-  extends SharedControllerDefinitionWithoutProps<Cart> {}
+  extends SearchAndListingControllerDefinitionWithoutProps<Cart> {}
```

Comment:
I believe search and context should be UniversalControllerDefinitionWithoutProps